client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -9,42 +9,25 @@ import {
   updateNewDirection,
 } from '../../api/socketService';
 import GamePage from '../Layout/GamePage';
-import {
-  boardCorners,
-  codeToEntity,
-  entityToCode,
-  getGhosts,
-  getPacman,
-  advanceFrameAfterTime,
-  boardTranspose,
-} from './constants';
-import {
-  initSquareGridState,
-  isWall,
-  getRandomAdjacentAvailableCell,
-  moveInDirection,
-  getGridwithWeights,
-  chaseLocation,
-} from './gameCore';
+import { arrowKeysDirections } from './constants';
+import { getObjectDiffs } from './gameCore';
 import PacmanBoard from './PacmanBoard';
 
 class PacmanGame extends Component {
   state = {
-    pacman: {
-      x: 1,
-      y: 5,
-      direction: 'RIGHT',
+    pacmans: {
+
     },
     ghosts: [],
     score: 0,
     status: 0, // 0 - Not-started, 1 - Progress, 2 - Restart
     gridState: [],
-    config: {
-      refreshRate: advanceFrameAfterTime,
-    },
-    moveGhostsCount: 0,
-    scatterGhostspath: [],
-    scatterStart: 75,
+    // config: {
```

Comment:
Why is this section commented? Could have been removed

Reply:
Removed.